### PR TITLE
Switch from s3cmd to awscli

### PR DESCRIPTION
**Pull request by DieterReuter**
In order to support new AWS regions like "eu-central-1" you cannot use `s3cmd` anymore. 

With `s3cmd` you will get an error like `ERROR: S3 error: 400 (InvalidRequest): The authorization mechanism you have provided is not supported. Please use AWS4-HMAC-SHA256.` More details could be found here: http://stackoverflow.com/questions/26533245/the-authorization-mechanism-you-have-provided-is-not-supported-please-use-aws4.

You can also send all your ENV variables via an `.env` file as well, just use `--env-file=.env` while calling `docker run`.

Here is an example of an `.env-template` file:
```
nproc=NUM_OF_CPU
ACCESS_KEY=ACCESS
SECRET_KEY=SECRET
REGION_NAME=REGION
BUCKET_NAME=BUCKET
```

When you build your Docker Image with `docker build -t resin/node-arm .`, you can run a build even from CLI with
```bash
docker run --rm=true --env-file=.env resin/node-arm /build.sh 0.12.0
```

Note: in your build server you have to support the env `BUCKET_NAME` and `REGION_NAME` to get the build working.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,13 +2,10 @@ FROM ubuntu:14.04
...
+		&& apt-get install -y git lib32stdc++6 python python-pip build-essential lib32z1 wget ca-certificates --no-install-recommends
 
-# Install S3 tool
-RUN wget -O- -q http://s3tools.org/repo/deb-all/stable/s3tools.key | sudo apt-key add - \
-		&& wget -O/etc/apt/sources.list.d/s3tools.list http://s3tools.org/repo/deb-all/stable/s3tools.list \
-		&& apt-get -q update \
-		&& apt-get install -y s3cmd \
+# Install AWS CLI
+RUN pip install awscli \
 		&& apt-get clean \
```
Comment (Page-) [inline Dockerfile]: Can you change this to the equivalent command to delete the `pip` cache please? :)

Comment (DieterReuter) [inline Dockerfile]: I just tried to find, which dir is used as pip cache, but didn't see anything in the Docker container. Normally I would expect "~/.cache/pip", but this doesn't exists at all. Maybe you know the right dir?

**Comment by DieterReuter** [comment]
@Page- I just rebuild the Docker Image, and checked it in detail. Now, there're no unused or sensible files included and there's no pip cache used at all.

**Comment by Page-** [comment]
Thanks for doing this @DieterReuter :)

**Pull request merged**